Formula/unfs3.rb

```diff
@@ -1,8 +1,20 @@
 class Unfs3 < Formula
   desc "User-space NFSv3 server"
-  homepage "http://unfs3.sourceforge.net"
-  url "https://downloads.sourceforge.net/project/unfs3/unfs3/0.9.22/unfs3-0.9.22.tar.gz"
-  sha256 "482222cae541172c155cd5dc9c2199763a6454b0c5c0619102d8143bb19fdf1c"
+  homepage "https://unfs3.sourceforge.io"
+
+  # Temporarily using Karl Mikaelsson's branch for nanosecond timestamps
+  # on Mac OS High Sierra until such time as he makes an official release.
+  # See: https://github.com/codekitchen/dinghy/issues/269#issuecomment-391007703
+  url "https://github.com/derfian/unfs3.git", :branch => "higher-resolution-setattr"
+  version "0.9.23-pre"
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
```

Comment:
It looks like this `=> :build` syntax is out of date, I had to `brew install automake` manually to get this to work. We'll need to change it on both this block and the one you copied it from below, but I can do that later if you prefer.

Reply:
Ok, sorry about that. Let's get the `fsevents_to_vm` changes agreed and merged, and then you'll have to make a new version of `dinghy` that depends on it. At that point, I think it will be easier to finish this formula, since the underlying dinghy version will have been incremented, so no tricks will be necessary to make it into an upgrade with an upgraded dependency (I think!)